### ISSUE TITLE
Debug Console: Suggest old expressions from history across sessions

### ISSUE THREAD

**Issue by octref** [microsoft/vscode#87129]
In Chrome I have this handy feature:

![image](https://user-images.githubusercontent.com/4033249/70954323-97bd5a00-2021-11ea-9847-aaee2088cfe2.png)

In VS Code, let's say one time I typed a super long regex, such as `str.replace(/......./, 'foo')`. In the next session, I cannot type to complete it. Whereas in Chrome I can.

When debugging one usually runs a session many times and evaluate the same expressions. I don't want to type it each time.

**Comment by isidorn** [comment]
You can use the up / down feature to get to history (what you already typed).
For what you want here is to see the history also as part of the completion items. I think this makes sense, and VS Code could ideally append the history to the end of the completion items.
If we do this we should do it on the vscode side, so each debug adapter profits.
The items from history would get a special icon and always be put at the end.
Assigning to this milestone to investigate further

fyi @weinand @roblourens @connor4312

**Comment by octref** [comment]
Thanks for considering it.

> You can use the up / down feature to get to history (what you already typed).

This does not work across sessions. Chrome's history does, which is useful when I have to debug one thing for many times.

**Comment by isidorn** [comment]
@octref up / down work across sessions for me (both if I restart vscode, or start debugging again). Can you pleae file a new issue with steps if up / down history does not work across sessions for you. Thanks

**Comment by isidorn** [comment]
I have added this feature. Now we will always suggest suggestions from history. I have decided to use the text icon for them and to always push them to the end of the completion list. However the suggest widget may decide to re-arange them to the top.
Try it out and let me know how it works for you.

Adding verificaiton needed so we get this verified.

![Screenshot 2019-12-31 at 16 00 20](https://user-images.githubusercontent.com/1926584/71625168-ab062300-2be6-11ea-92ec-c5a9ff927ed4.png)
